### PR TITLE
docs: Update upgrade docs

### PR DESCRIPTION
**Pull request by ty-dc**
#### What type of PR is this?

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/usage/install/upgrade-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+# 升级指南
+
+本升级指南适用于在 Kubernetes 上运行的 Spiderpool。如果您有任何疑问，请随时通过 [Spiderpool Community](../../README-zh_CN.md#社区) 联系我们。
```
Comment (windsonsea) [inline docs/usage/install/upgrade-zh_CN.md:3]:
```suggestion
本升级指南适用于在 Kubernetes 上运行的 Spiderpool。如果您有任何疑问，请随时通过 [Spiderpool Community](../../README-zh_CN.md#_6) 联系我们。
```

File: docs/usage/install/upgrade-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    docker pull ghcr.io/spidernet-io/spiderpool/spiderpool-controller:[upgraded-version]
+
+    # 如果您是中国大陆用户，可以使用镜像源 ghcr.m.daocloud.io
+    docker pull ghcr.m.daocloud.io/spidernet-io/spiderpool/spiderpool-agent:[upgraded-version]
+    docker pull ghcr.m.daocloud.io/spidernet-io/spiderpool/spiderpool-controller:[upgraded-version]
+    ```
+
+## 步骤
+
+建议每次都升级到 Spiderpool 的最新且被维护的补丁版本。通过 [Stable Releases](../../README-zh_CN.md#稳定版本) 了解受到支持的最新补丁版本。
```
Comment (windsonsea) [inline docs/usage/install/upgrade-zh_CN.md:23]:
```suggestion
建议每次都升级到 Spiderpool 的最新且被维护的补丁版本。通过 [Stable Releases](../../README-zh_CN.md#_2) 了解受到支持的最新补丁版本。
```

File: docs/usage/install/upgrade-zh_CN.md (hunk 1 above)
Comment (windsonsea) [reply docs/usage/install/upgrade-zh_CN.md:3]: mkdocs 不支持中文锚点吧

Comment (ty-dc) [reply docs/usage/install/upgrade-zh_CN.md]: done

File: docs/usage/install/upgrade-zh_CN.md (hunk 2 above)
Comment (ty-dc) [reply docs/usage/install/upgrade-zh_CN.md]: done

**Review by weizhoublue — APPROVED** [review]

**Review by Icarus9913 — APPROVED** [review]

**Pull request merged**